<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelSettingDialog</class>
 <widget class="QWidget" name="LabelSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LabelSettingDialog</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>20</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Label Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_thickness">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>85</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Label Thickness</string>
   </property>
  </widget>
  <widget class="QLabel" name="labe_color">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>55</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Label Color</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>input labelling name ( ex : car )</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_btn">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>115</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancle_btn">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>115</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancle</string>
   </property>
  </widget>
  <widget class="QComboBox" name="color_box">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="thickness_box">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancle_btn</sender>
   <signal>pressed()</signal>
   <receiver>LabelSettingDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
